testing/trino-product-tests/src/main/java/io/trino/tests/product/kudu/TestKuduConnectoKerberosSmokeTest.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.tests.product.kudu;
+
+import io.trino.tempto.query.QueryResult;
+import org.testng.annotations.Test;
+
+import java.util.UUID;
+
+import static io.trino.tempto.assertions.QueryAssert.Row.row;
+import static io.trino.tempto.assertions.QueryAssert.assertThat;
+import static io.trino.tests.product.TestGroups.KUDU;
+import static io.trino.tests.product.utils.QueryExecutors.onTrino;
+import static java.lang.String.format;
+
+public class TestKuduConnectoKerberosSmokeTest
+{
+    @Test(groups = KUDU)
+    public void kerberosAuthTicketExpiryTest() throws InterruptedException
+    {
+        String kuduTable = "kudu.default.nation_" + UUID.randomUUID().toString().replace("-", "");
+        String table = "tpch.tiny.nation";
+
+        assertThat(onTrino().executeQuery(format("SELECT count(*) from %s", table))).containsExactlyInOrder(row(25));
+        QueryResult result = onTrino().executeQuery(format("CREATE TABLE %s AS SELECT * FROM %s", kuduTable, table));
+        try {
+            assertThat(result).updatedRowsCountIsEqualTo(25);
+            assertThat(onTrino().executeQuery(format("SELECT count(*) FROM %s", kuduTable))).containsExactlyInOrder(row(25));
+            // Kerberos tickets are configured to expire after 60 seconds, this should expire the ticket
+            Thread.sleep(120_000L);
```

Comment:
can we make it shorter? (configurable)? Can we expire it programmatically

Reply:
I configured the default ticket expiry in tests to be 1 min, see `kerberos_init.sh`, I can make this lower just wanted to be very safe. 

I'll look a bit more into `useTicketCache` in JAAS, it is possible we can force jaas to use the ticket cache then destroy the ticket cache programmatically
https://docs.oracle.com/javase/8/docs/jre/api/security/jaas/spec/com/sun/security/auth/module/Krb5LoginModule.html

Reply:
When I tried to configure `useTicketCache` with debug logs on, it attempted to read a ticket cache file that did not exist. 

It looks like the java kerberos client implementation does not ever write tickets to the ticket cache, it only supports *reading* from an existing ticket cache, see here for some more details:
https://stackoverflow.com/questions/45463635/jaas-fails-to-persist-kerberos-ticket-to-cache-file-and-unable-to-create-cach/45618313#45618313

Going down that path doesn't seem promising.

I'm not a kerberos expert, if other people have ideas let me know! But it sounds like revocation of kerberos tickets is not usually done because ticket lifetimes are so short.